src/main/resources/assets/admin/common/js/security/PrincipalLoader.ts

```diff
@@ -65,6 +65,11 @@ export class PrincipalLoader
         return this.getRequest().isPartiallyLoaded();
     }
 
+    setUsePreData(bool: boolean): PrincipalLoader {
+        super.setUsePreData(bool);
+        return this;
```

Comment:
I initially tried without that before adding this, but it is necessary in order to avoid the following type error:

```bash
ERROR in /home/bruno/Enonic/Work/app-users/src/main/resources/assets/js/app/wizard/MembersWizardStepForm.ts
./js/app/wizard/MembersWizardStepForm.ts 27:8-19
[tsl] ERROR in /home/bruno/Enonic/Work/app-users/src/main/resources/assets/js/app/wizard/MembersWizardStepForm.ts(27,9)
      TS2740: Type 'BaseLoader<Principal>' is missing the following properties from type 'PrincipalLoader': skipPrincipalKeys, setIdProviderKey, setAllowedTypes, skipPrincipals, and 6 more.
ts-loader-default_a99870468de4773e
 @ ./js/app/wizard/MembersWizardPanel.ts 22:30-64
 @ ./js/app/wizard/RoleWizardPanel.ts 19:27-58
 @ ./js/app/UserAppPanel.ts 28:24-59
 @ ./js/main.ts 39:21-50
```

The thing is that the return of them are different since the `this` is different in each file.